bundles/org.openhab.binding.broadlink/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
binding.xml is no longer used since openHAB 4.x Please remove.

Reply:
Done